stage0/src/zero_page.rs

```diff
@@ -148,19 +148,11 @@ impl ZeroPage {
         self.inner.e820_entries = e820_entries as u8;
         self.validate_e820_table();
 
-        // Carve out a chunk of memory out for the ACPI area and reserved memory just
-        // below 1 MiB. The ACPI area is from 0x80000-0x9FFFF. We also reserve
-        // the region from 0xA0000-0xEFFFF since historically this contained
+        // Carve out a chunk of memory out for the ACPI area from 0x80000-0x9FFFF. We
```

Comment:
Not sure I am parsing this comment correctly. Which of these does it mean:

- (from 0x80000-0x9FFFF) (to something else)
- (from 0x80000) (to 0x9FFFF)

If the latter, I think the syntax is incorrect -- either remove the "from", or add a "to".

Also it seems here the upper bound is inclusive, but in the code it is exclusive. Maybe clarify either way.

Reply:
Replace with an exclusive interval in the comments.